sage-android/Sage/app/src/main/java/blueprint/com/sage/shared/views/Spacer.java

```diff
@@ -0,0 +1,24 @@
+package blueprint.com.sage.shared.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.View;
+
+/**
+ * Created by charlesx on 10/13/15.
+ * Adds spacing -- is this hacky?
+ */
+public class Spacer extends View {
```

Comment:
I guess if it's necessary, its not a bad thing to have, but what's the advantage over margins / padding?